coralproject/talk
# Q&A Beta

## What does this do?

Adds full Q&A features to the stream.

Figma Spec: https://www.figma.com/file/IJu2JNZFNvYiolJWEdNeY7al/Talk-v5---Stream?node-id=4211%3A0

## How to test?

- Enable Q&A using the new configure tab
- Add an expert
- Go back to the stream and ask questions as a commenter
- Respond to them as the expert user
- See how the functionality works with the 3 new tabs
